### DISCUSSION TITLE
Drop "in this module" from Learn?

### DISCUSSION THREAD

**Discussion by wbamberg** [mdn/mdn-community#328] [Content] [answered]
In the Learn docs, every "lesson page" has a section at the end titled "In this module", that lists all the lessons in this module: https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Build_your_own_function#in_this_module.

This list is very hard to maintain, because every time a new lesson is added, removed, or even just reordered, we have to update this section in every page, and people always forget or make mistakes.

Over in https://github.com/mdn/yari/pull/7444#issuecomment-1296438768, @hamishwillee suggested that now sidebars are less broken, perhaps we don't need this any more, and the same thing came up in https://github.com/mdn/yari/pull/8014#issuecomment-1425423981.

So this discussion is to find out if there is consensus for this.

**Comment by hamishwillee** [comment]
Yes please.

To be very clear I think the module lists provide a clearer picture that the modules are a set of related and ordered topics than the sidebar. If it were possible/easy to generate them from the sidebar I'd be happy with that.

But I don't think they are needed any more to provide general context of what comes before and next, and IMO they are certainly not worth the cost of maintenance.

**Comment by bsmth** [comment]
I would add a `+1` for removing them, also. Thanks for summarizing and proposing the changes 👍🏻

**Comment by chrisdavidmills** [comment]
To provide a bit of context here, the only reason these were added is because Sheppy was doing some SEO research way back when, and one of the items that came up is that each page should have a clear TOC for the directory it is in, so you've got a clear navigation path across the directory, regardless of what level you are at. 

But I'm not sure how much advantage this brings in terms of SEO, and I doubt it outweighs the maintenance burden. So I would support this too, provided we make sure readers still have their prev and next links, so the learning pathway is not broken.

**Reply by estelle** [reply]
There definitely should be the list on the initial page, as a TOC, but not on every page.

**Reply by wbamberg** [reply]
Thanks for the context, Chris! FWIW I don't feel very strongly about this either way. Like most people probably, I think they are somewhat useful but wish we had a more maintainable way of doing things like this. And yes, this won't affect next/previous.

**Reply by chrisdavidmills** [reply]
> Thanks for the context, Chris! FWIW I don't feel very strongly about this either way. Like most people probably, I think they are somewhat useful but wish we had a more maintainable way of doing things like this. And yes, this won't affect next/previous.

Perfect, I think we can get rid of them then. If a strong argument for reinstating them ever comes up in the future, we can spend some time implementing an automated way to include them instead.

**Comment by Rumyra** [comment]
Sounds like there's consensus for this to go ahead - if we feel it's necessary to have an overview we should find a way to generate it and not author it 👍

**Comment by wbamberg** [comment, marked as answer]
I've filed an issue to remove it -> https://github.com/mdn/content/issues/24467
